docker/test/docker-compose.yml

```diff
@@ -10,8 +10,8 @@ services:
     ports:
       - 8080:8080
     volumes:
-      - ./logs/:/opt/orbs/logs
-      - ./e2e-config/:/opt/orbs/config
+      - ../../logs/:/opt/orbs/logs
+      - ../../e2e-config/:/opt/orbs/config
```

Comment:
let's find a better place for ./e2e-config` in the project root, maybe `./test/e2e/_config`

Again, the idea of the underscore convention is to indicate that it's not go source files

Reply:
Moved to `docker/test/e2e-config`.